presto-pinot/src/main/java/io/prestosql/pinot/client/PinotClient.java

```diff
@@ -94,10 +92,8 @@
     private static final String QUERY_URL_TEMPLATE = "http://%s/query/sql";
 
     private final List<String> controllerUrls;
-    private final PinotMetrics metrics;
```

Comment:
It looks like `PinotMetrics` class is mostly unused except for two utility methods (`doWithRetries`, `isValidPinotHttpResponseCode`).

Reply:
Should we keep other unused methods ?
If we need to remove those unused methods , `PinotMetrics` and `PinotMetricsStats` could be removed (and move `doWithRetries` to `PrestoClient`).

Reply:
Correct, no need to keep the unused methods.